diff --git a/sys/kern/uipc_sem.c b/sys/kern/uipc_sem.c
index 094979517..482281197 100644
--- a/sys/kern/uipc_sem.c
+++ b/sys/kern/uipc_sem.c
@@ -87,8 +87,8 @@ MODULE(MODULE_CLASS_MISC, ksem, NULL);
 
 #define	SEM_MAX_NAMELEN		NAME_MAX
 
-# https://muc.lists.netbsd.bugs.narkive.com/2z13sHNT/kern-53998-sem-init-fails-with-error-1
-# allow increase of SEM_NSEMS_MAX (incorrect error reported, see above)
+// https://muc.lists.netbsd.bugs.narkive.com/2z13sHNT/kern-53998-sem-init-fails-with-error-1
+// allow increase of SEM_NSEMS_MAX (incorrect error reported, see above)
 #ifndef SEM_NSEMS_MAX
 #define	SEM_NSEMS_MAX		256
 #endif
